app/src/main/java/org/oppia/app/activity/InputInteractionViewTestActivity.kt

```diff
@@ -7,7 +7,7 @@ import org.oppia.app.customview.interaction.NumericInputInteractionView
 
 /**
  * This is a dummy activity to test input interaction views.
- * It contains [NumericInputInteractionView] .
+ * It contains [NumericInputInteractionView] , [TextInputInteractionView]  and  [FractionInputInteractionView].
```

Comment:
Spacing issue: 
`* It contains [NumericInputInteractionView], [TextInputInteractionView] and [FractionInputInteractionView].`

Reply:
Also, import these two:

`import org.oppia.app.customview.interaction.TextInputInteractionView`
`import org.oppia.app.customview.interaction.FractionInputInteractionView`

Reply:
Updated